easybuild/easyconfigs/m/MixClone/MixClone-1.1.6.eb

```diff
@@ -0,0 +1,47 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'PythonBundle'
+
+name = 'MixClone'
+version = '1.1.6'
+
+homepage = 'https://github.com/uci-cbcl/MixClone#usage'
+description = """MixClone is a comprehensive software package to study the subclonal 
+structures of tumor genomes, including subclonal cellular prevalences estimation, allelic 
+configuration estimation, absolute copy number estimation and a few visualization 
+tools."""
+
+toolchain = SYSTEM  # or = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '2.7.16-GCCcore-8.3.0'),
+    ('SciPy-bundle', '2019.10-foss-2019b-Python-2.7.16'),
+]
+
+use_pip = True
+
+# All additional Python packages that are required, our package must be last in list 
+exts_list = [
+    ('pysam', '0.10.0', {
```

Comment:
We have a separate easyconfig for `Pysam` (`Pysam-0.16.0.1-GCC-8.3.0.eb`), so it's better to include a proper dependency for `Pysam` instead (and then switch to `PythonPackage` rather than `PythonBundle`)

Unless there's a specific reason you're using an older version of `pysam`?

Reply:
The MixClone code is like 5 years old so I am trying to use a version of pysam from that bygone era to minimize compatibility issues